From 326399ceef0c904f78cdd5da17f7ca724292b4f8 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 7 Aug 2023 10:36:46 +0100
Subject: [PATCH 3/3] build: Go back to gettext 0.19
 
Commit 850feece ("build: Remove gnulib") unintentionally(?) moved to
GNU gettext 0.21.  However this is not available in RHEL 8, and
doesn't contain any new features we need.  Go back to 0.19 from RHEL 8.
 
Update: commit 850feece47b052e568392e83bff2bcdb2886f14d
---
 configure.ac         |  2 +-
 po/Makevars.template | 29 -----------------------------
 2 files changed, 1 insertion(+), 30 deletions(-)
 
diff --git a/configure.ac b/configure.ac
index 946c261..7a8b664 100644
--- a/configure.ac
+++ b/configure.ac
@@ -136,7 +136,7 @@ AS_IF([test "x$with_readline" != xno],
 
 dnl For i18n.
 AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.21])
+AM_GNU_GETTEXT_VERSION([0.19])
 AM_ICONV
 
 dnl libxml2 (optional).
diff --git a/po/Makevars.template b/po/Makevars.template
index 86a11f1..4a9ff7d 100644
--- a/po/Makevars.template
+++ b/po/Makevars.template
@@ -1,8 +1,4 @@
 # Makefile variables for PO directory in any package using GNU gettext.
-#
-# Copyright (C) 2003-2019 Free Software Foundation, Inc.
-# This file is free software; the Free Software Foundation gives
-# unlimited permission to use, copy, distribute, and modify it.
 
 # Usually the message domain is the same as the package name.
 DOMAIN = $(PACKAGE)
@@ -24,13 +20,6 @@ XGETTEXT_OPTIONS = --keyword=_ --keyword=N_
 # their copyright.
 COPYRIGHT_HOLDER = Free Software Foundation, Inc.
 
-# This tells whether or not to prepend "GNU " prefix to the package
-# name that gets inserted into the header of the $(DOMAIN).pot file.
-# Possible values are "yes", "no", or empty.  If it is empty, try to
-# detect it automatically by scanning the files in $(top_srcdir) for
-# "GNU packagename" string.
-PACKAGE_GNU =
-
 # This is the email address or URL to which the translators shall report
 # bugs in the untranslated strings:
 # - Strings which are not entire sentences, see the maintainer guidelines
@@ -62,21 +51,3 @@ USE_MSGCTXT = no
 #   --previous            to keep previous msgids of translated messages,
 #   --quiet               to reduce the verbosity.
 MSGMERGE_OPTIONS =
-
-# These options get passed to msginit.
-# If you want to disable line wrapping when writing PO files, add
-# --no-wrap to MSGMERGE_OPTIONS, XGETTEXT_OPTIONS, and
-# MSGINIT_OPTIONS.
-MSGINIT_OPTIONS =
-
-# This tells whether or not to regenerate a PO file when $(DOMAIN).pot
-# has changed.  Possible values are "yes" and "no".  Set this to no if
-# the POT file is checked in the repository and the version control
-# program ignores timestamps.
-PO_DEPENDS_ON_POT = yes
-
-# This tells whether or not to forcibly update $(DOMAIN).pot and
-# regenerate PO files on "make dist".  Possible values are "yes" and
-# "no".  Set this to no if the POT file and PO files are maintained
-# externally.
-DIST_DEPENDS_ON_UPDATE_PO = yes
-- 
2.41.0
 